book-library/script.js

```diff
@@ -89,12 +91,12 @@ function render() {
     });
 
     //add delete button to every row and render again
-    let delButton = document.createElement("button");
-    delBut.id = i + 5;
+    let delBut = document.createElement("button");
+    delBut.id = "d" + i;
```

Comment:
when creating read button it was assigning "i" as id and here it was assigning "i + 5" as id. after 5ft element, id"s of read button and delete button was overlapping. solved that problem with adding "d" before "I"